### ISSUE TITLE
Which configuration are you using?

### ISSUE THREAD

**Issue by poojashresthacode** [ubsuny/23-Homework7G1#16]

**Comment by tirthbha** [comment]
@poojashresthacode I wil do -6.8696 and -6.7607

**Comment by LinxuanHu** [comment]
> @poojashresthacode I wil do -6.8696 and -6.7607

i might get it wrong, but there are two figures in -6.7607, not sure if they are same or different configurations.

**Comment by poojashresthacode** [comment]
I think they are different configuration with different B.E.
________________________________

**Comment by poojashresthacode** [comment]
Sorry by.mistake, I mean to say different configuration with same B.E
________________________________

**Comment by tirthbha** [comment]
@LinxuanHu Yes, they are two different structures with same binding energy.
